aspnetcore/signalr/httpcontext.md

```diff
@@ -0,0 +1,23 @@
+---
+title: IHttpContextAccessor/HttpContext in ASP.NET Core SignalR
+author: guardrex
+description: Learn about IHttpContextAccessor and HttpContext in ASP.NET Core SignalR.
+monikerRange: '>= aspnetcore-3.1'
+ms.author: riande
+ms.custom: mvc
+ms.date: 01/30/2025
+uid: signalr/httpcontext
+---
+# `IHttpContextAccessor`/`HttpContext` in ASP.NET Core SignalR
```

Comment:
For SignalR you should use `Context.GetHttpContext()` when in the hub. See https://learn.microsoft.com/aspnet/core/signalr/hubs?view=aspnetcore-9.0#the-context-object for details.

Reply:
Thanks @BrennanConroy ... See if my last commit places a good paragraph for it.

Reply:
@BrennanConroy ... pinging again in case you didn't see the earlier one.